web/controllers/chatNotificationController/index.ts

```diff
@@ -0,0 +1,46 @@
+import { getLogger } from '../../../common/logger/logger';
+import { Request, Response, Router } from 'express';
+import { NotificationTriggered } from './types';
+import { talkJsIdToUserId } from '../../../common/chat/helper';
+import { getPupil, getStudent, getUser } from '../../../common/user';
+import * as Notification from '../../../common/notification';
+import { pupil as Pupil, student as Student } from '@prisma/client';
+import { ChatType, getChatType, getNotificationContext, verifyChatUser } from './util';
+
+const logger = getLogger('ChatNotification');
+
+export const chatNotificationRouter = Router();
+
+chatNotificationRouter.post('/chat-notification', handleChatNotification);
+
+async function handleChatNotification(req: Request, res: Response): Promise<void> {
+    try {
```

Comment:
missing authentication of the other side!

Reply:
Can you tell me more specifically what you mean by that?

Reply:
How do you know the request comes from TalkJS?

Reply:
https://talkjs.com/docs/Features/Security_Settings/Security_Recommendations/#webhook-integrity